.github/CODEOWNERS

```diff
@@ -562,47 +562,24 @@
 #/<NotInRepo>/                                                     @kpiteira
 
 # ServiceLabel: %Monitor %Service Attention
-/sdk/monitor/                                                      @SameergMS @dadunl
-/sdk/monitor/ci.yml                                                @nisha-bhatia @JoshLove-msft @pallavit @Azure/azure-sdk-write-monitor-data-plane
+#/<NotInRepo>/                                                     @SameergMS @dadunl @AzMonEssential @AzmonAlerts @AzmonActionG @AzmonLogA
```

Comment:
I believe these groups need the org prefix.   For example: `@Azure/AzmonAlerts`

Reply:
I thought the same thing initially. These appear to be GitHub accounts, rather than teams.

Reply:
huh... well, ok then.   😄